File: NewProject/Models/SPYWXHZY.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;


namespace NewProject.Models
{
    //药物相互作用
    public class SPYWXHZY : Table
    {
        [Key]
        public int GUID { get; set; }
        [Display(Name = "相互作用ID")]
        public string XHZYBH { get; set; }
        [Display(Name = "产品ID1")]
        public int SPBH1 { get; set; }
        [Display(Name = "产品ID2")]
        public int SPBH2 { get; set; }
        [Display(Name = "作用效果")]
        public string ZYXG { get; set; }
    }
}

File: NewProject/wwwroot/js/Class/Spywxhzy.js
//商品药物相互作用
export default class Spywxhzy {
    constructor() {
        //相互作用ID
        this.xhzybh = "";
        //产品ID1
        this.spbh1 = "";
        //产品ID2
        this.spbh2 = "";
        //作用效果
        this.zyxg = "";
    }
    addData(obj) {
        this.xhzybh = obj.xhzybh;
        this.spbh1 = obj.spbh1;
        this.spbh2 = obj.spbh2;
        this.zyxg = obj.zyxg;
    }
}